Specs/Number/Japanese/NumberRangeModel.json

```diff
@@ -322,5 +342,1785 @@
         "End": 6
       }
     ]
+  },
+  {
+    "Input": "この数は２０より大きく３５以下です。",
+    "IgnoreResolution": true,
+    "NotSupportedByDesign": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "20より大きく35以下",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(20,35]"
+        },
+        "Start": 4,
+        "End": 14
+      }
+    ]
+  },
+  {
+    "Input": "この数は２０以上３０未満の間にあります。",
+    "IgnoreResolution": true,
+    "NotSupportedByDesign": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "20以上30未満の間",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "[20,30)"
+        },
+        "Start": 4,
+        "End": 13
+      }
+    ]
+  },
+  {
+    "Input": "彼は１０番目以上１５番目未満の間に位置付けされます。",
+    "IgnoreResolution": true,
+    "NotSupportedByDesign": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "10番目以上15番目未満の間",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "[10,15)"
+        },
+        "Start": 2,
+        "End": 15
+      }
+    ]
+  },
+  {
+    "Input": "彼は−１０以上１５未満の間を得る。",
+    "IgnoreResolution": true,
+    "NotSupportedByDesign": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "-10以上15未満の間",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "[-10,15)"
+        },
+        "Start": 2,
+        "End": 12
+      }
+    ]
+  },
+  {
+    "Input": "この数は１００より大きく３００より小さいです。",
+    "IgnoreResolution": true,
+    "NotSupportedByDesign": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "100より大きく300より小さい",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "-100,300"
```

Comment:
It seems some of the specs are not fully complete. Please double check values.
By default, every interval is closed at the start (includes the starting point) and open at the end (does not include the end point). Modifiers will change this behaviour. Please look into English and Chinese for examples.

Reply:
Re-validated and made changes